packages/core/src/Link.js

```diff
@@ -13,6 +14,13 @@ const Link = styled.a`
   ${applyVariations('Link')}
 `
 
+const Link = props => {
```

Comment:
nb: you can also represent this like:
```
const Link = styled.a.attrs(props => ({
  rel: props.target === '_blank' ? 'noopener' : null
}))`
  cursor: pointer;
  text-decoration: none;
  color: ${getPaletteColor('base')};
  ${applyVariations('Link')}
`
```

Reply:
Oh that's way cleaner, thank you! Replacement has been pushed up.